seb_openedx/dashboard/views.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""The generic views for the exc-core plugin project"""
+
+from __future__ import unicode_literals
+import django_tables2 as tables
```

Comment:
What do we get from this django_tables2 package? for this to be installed in production we must go out of the regular way, so this requirement better be super worth it

Reply:
Ok; I'll hardcode the HTML instead then.